### ISSUE TITLE
Header - Windows High Contrast Mode problem with text in SVG

### ISSUE THREAD

**Issue by iamjolly** [18F/samhsa-prototype#312]
### Problem Description
In Windows High Contrast Mode (Win10, Edge), the Public Sans text color (dark gray) within the SVG which reads "TreatmentFinder.gov" does not honor Windows High Contrast settings, so it becomes almost invisible against the default black background color (screenshot with area circled).

![samhsa-home-ms-edge-win10-HCM](https://user-images.githubusercontent.com/1093423/64820313-5a9e8300-d56d-11e9-97b6-3f58f57446d0.png)

### Recommended Solution
Remove text from SVG and use text rendered on the page using the Public Sans webfont. This will allow WHCM to change the color of the text to something that works with the user's settings.

**Comment by iamjolly** [comment]
After talking with a colleague today about contrast, I realized there is another possible solution to this issue... a white background could be added to the TreatmentFinder.gov SVG, and it would appear as-is in Windows High Contrast Mode.

**Comment by iamjolly** [comment]
Resolved via https://github.com/18F/samhsa-prototype/commit/1a1c28421508b42962fac7aa2085f60b25ea18f8 - made white background for SVG for now.
